pkg/api/common/versions.go

```diff
@@ -92,8 +92,8 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.12.0":         false,
 	"1.12.1":         true,
 	"1.12.2":         true,
-	"1.13.0-alpha.1": true,
-	"1.13.0-alpha.2": true,
+	"1.13.0":         false,
```

Comment:
copying my comment over from the other PR: Adding "1.13.0" would imply that it was supported at some point in time which it wasn't but also it wouldn't have any functional impact to add it so I'm okay with either way.

Reply:
I agree with your argument--I'm removing it.